Code/Engine/RendererCore/Debug/Implementation/DebugRenderer.cpp

```diff
@@ -2100,35 +2100,45 @@ EZ_BEGIN_STATIC_REFLECTED_TYPE(ezScriptExtensionClass_Debug, ezNoBase, 1, ezRTTI
   {
     EZ_SCRIPT_FUNCTION_PROPERTY(DrawCross, In, "World", In, "Position", In, "Size", In, "Color", In, "Transform")->AddAttributes(
       new ezFunctionArgumentAttributes(2, new ezDefaultValueAttribute(0.1f)),
-      new ezFunctionArgumentAttributes(3, new ezExposeColorAlphaAttribute())),
+      new ezFunctionArgumentAttributes(3, new ezDefaultValueAttribute(ezColor::White), new ezExposeColorAlphaAttribute()),
```

Comment:
I don't think that these default value attribute should be needed, for properties you also don't need to specify them when the have the type's default value. Should be the same here.

Reply:
They are needed when auto generating function signatures in languages that work like C, ie when argument N has a default value, then argument N+1 also has to have a default value.

Also, for these functions they do make sense to have.

Reply:
removed some of them again